### ISSUE TITLE
[Docs] Document on How to write new distro chroot config in mock

### ISSUE THREAD

**Issue by TommyLike** [rpm-software-management/mock#961]
### Short description of the problem
I am from another linux distro community called openEuler [1], Is there any document on how to write the chroot config and finally add it into mock project?

[1]: https://www.openeuler.org/en/

**Comment by xsuchy** [comment]
https://docs.pagure.org/copr.copr/how_to_manage_chroots.html

I will highlight:

Copr uses [Mock](https://github.com/rpm-software-management/mock) for building packages, so you should check if the mock configs [are already available](https://github.com/rpm-software-management/mock/tree/devel/mock-core-configs/etc/mock) and in which version of the mock-core-configs package they were added. This package needs to be on the Copr builders.

In other words - if you want to create new chroot e.g., openeuler-1-x86_64 you must **somehow**  arrange that builders have the file /etc/mock/openeuler-1-x86_64.cfg

It is up to you if you just copy it on provision or if you add it to upstream of mock-core-configs.

**Comment by TommyLike** [comment]
> https://docs.pagure.org/copr.copr/how_to_manage_chroots.html
> 
> I will highlight:
> 
> Copr uses [Mock](https://github.com/rpm-software-management/mock) for building packages, so you should check if the mock configs [are already available](https://github.com/rpm-software-management/mock/tree/devel/mock-core-configs/etc/mock) and in which version of the mock-core-configs package they were added. This package needs to be on the Copr builders.
> 
> In other words - if you want to create new chroot e.g., openeuler-1-x86_64 you must **somehow** arrange that builders have the file /etc/mock/openeuler-1-x86_64.cfg
> 
> It is up to you if you just copy it on provision or if you add it to upstream of mock-core-configs.

Thanks @xsuchy , assume I need to contribute new chroot config, let's be more specific:
1. what is the meaning of this config options? [1](https://github.com/rpm-software-management/mock/blob/main/mock-core-configs/etc/mock/templates/anolis-8.tpl#L9), I can guess some of them, but not all of them.
2. what is required for upstream our own version of chroot config? Is there any CI gate to ensure it always work?
